youtube_dl/extractor/streamme.py

```diff
@@ -0,0 +1,193 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    ExtractorError,
+)
+
+
+class StreamMeIE(InfoExtractor):
+    IE_NAME = 'StreamMe:video'
+    _API_CHANNEL = 'https://www.stream.me/api-user/v1/<channel_id>/channel'
+    _API_ARCHIVE = 'https://www.stream.me/api-vod/v1/<channel_id>/archives'
+    _VALID_URL_BASE = r'https?://(video-cdn|www).stream.me'
+    _VALID_URL = r'%s\/archive\/(?P<channel_id>[^\#\/]+)\/[^\/]+\/(?P<id>[^\/]+)' % _VALID_URL_BASE
+    _TEST = {
+        'url': 'https://www.stream.me/archive/kombatcup/kombat-cup-week-8-sunday-open/pDlXAj6mYb',
+        'md5': 'b32af6fad972d0bcf5854a416b5b3b01',
+        'info_dict': {
+            'id': 'pDlXAj6mYb',
+            'ext': 'mp4',
+            'title': 'Kombat Cup Week #8 - Sunday Open',
+            'uploader': 'KombatCup',
+            'uploader_id': 'kombatcup',
+            'timestamp': 1481512102000,
+            'age_limit': 13,
+        }
+    }
+
+    def _real_extract(self, url):
+        m = re.match(self._VALID_URL, url)
+        video_id = self._match_id(url)
+        apiurl = self._API_ARCHIVE.replace('<channel_id>', m.group('channel_id'))
+
+        data = json.loads(self._download_webpage(apiurl, video_id))
+
+        for vod in data.get('_embedded').get('vod'):
```

Comment:
There is no point in `get()` here. You will fail anyway if there is no corresponding keys in dict.